Scenes/MainMenu/MainMenu.tscn

```diff
@@ -418,6 +419,11 @@ margin_top = -40.0
 margin_right = 62.0
 text = "Back"
 
+[node name="AudioStreamPlayer" type="AudioStreamPlayer" parent="."]
```

Comment:
Somehow I missed that we already have a music player on this scene on the main branch. If you wanna cancel this PR that's fine! Just want to here that awesome intro music when the MainMenu loads! :D

Reply:
Haha! I added it in not long after merging in the other branch. We were on the same wavelength there :+1: